client/python/examples/simple.py

```diff
@@ -0,0 +1,90 @@
+"""
+Example of using the Armada client to create a queue, jobset and job,
+then watch for the job to succeed or fail.
```

Comment:
This is the sort of information that should be above, where you're talking about what examples we have. IMO